frontend/src/component/user/Profile/ProfileTab/ProfileTab.tsx

```diff
@@ -131,7 +131,7 @@ export const ProfileTab = ({ user }: IProfileTabProps) => {
                 <StyledAvatar user={user} />
                 <StyledInfo>
                     <StyledInfoName>
-                        {user.name || user.username}
+                        {user.name || user.email || user.username}
```

Comment:
Scouting. Noticed this had the same issue: A user without name or username (email only) would show undefined here.